src/zkevm_specs/evm/execution/call_staticcall.py

```diff
@@ -78,6 +86,9 @@ def call(instruction: Instruction):
     has_value = 1 - instruction.is_zero(value)
     instruction.constrain_zero(has_value * is_static)
 
+    # Constrain value == 0 for opcode STATICCALL.
+    instruction.constrain_zero(has_value * (1 - is_call))
```

Comment:
It seems line 87 is already doing this.

Reply:
Deleted this constraint and reserve `constrain_zero(has_value * is_static)` above.